buildlib/azure-pipelines.yml

```diff
@@ -114,27 +117,27 @@ stages:
               set -e
               mkdir build-arm64
               cd build-arm64
-              CC=aarch64-linux-gnu-gcc-8 cmake -GNinja .. -DIOCTL_MODE=both -DNO_PYVERBS=1 -DENABLE_WERROR=1
+              CC=aarch64-linux-gnu-gcc-9 cmake -GNinja .. -DIOCTL_MODE=both -DNO_PYVERBS=1 -DENABLE_WERROR=1
```

Comment:
Can you remind me why aren't we using latest gcc-10 for aarch64 and ppc?

Reply:
ubuntu doesn't build the cross compilers for the gcc ppa